relationships/synthesis/APM-APPLICATION-to-KAFKATOPIC.yml

```diff
@@ -120,13 +120,20 @@ relationships:
         extractGuid:
           attribute: entityGuid
       target:
-        lookupGuid:
-          candidateCategory: KAFKATOPIC
-          fields:
-            - field: clusterName
-              attribute: clusterName
-            - field: topic
-              attribute: topic
+        buildGuid:
```

Comment:
these are breaking changes. But I also see that you did them a few days ago. Hence we are moving forward with them